lisp/forge-post.el

```diff
@@ -98,7 +98,7 @@
 (defvar-local forge--cancel-post-function nil)
 (defvar-local forge--pre-post-buffer nil)
 
-(cl-defun forge--prepare-post-buffer (filename repo)
+(cl-defun forge--prepare-post-buffer (filename repo &key header)
```

Comment:
Do you expect to add more arguments or why did you choose to use a keyword argument?

What do you think about making `repo` the first argument?

Reply:
`repo` being the first argument: I'm a fan; makes sense. I've made this update.

I expect more arguments might be needed over time, but this might be a coding style precaution carried over from work (where function signatures – even private ones – are rarely changed). I can promote it to a formal, optional argument, but I do think keys provide some readability benefits.